pip install on OS X failing

Hi,

I'm running into some trouble installing `scikit-bio`, similar to #981.

When I run `$ pip3 install scikit-bio` I get a lot of output, the main part being building the wheel fails and then building from source fails, both ending with the following error:

```
   building 'skbio.stats.__subsample' extension
    creating build/temp.macosx-10.11-x86_64-3.5
    creating build/temp.macosx-10.11-x86_64-3.5/skbio
    creating build/temp.macosx-10.11-x86_64-3.5/skbio/stats
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/include -I/usr/local/opt/openssl/include -I/usr/local/opt/sqlite/include -I/usr/local/Cellar/python3/3.5.2_1/Frameworks/Python.framework/Versions/3.5/include/python3.5m -c skbio/stats/__subsample.c -o build/temp.macosx-10.11-x86_64-3.5/skbio/stats/__subsample.o
    skbio/stats/__subsample.c:250:10: fatal error: 'numpy/arrayobject.h' file not found
    #include "numpy/arrayobject.h"
             ^
    1 error generated.
    error: command 'clang' failed with exit status 1
 ```

I'm running python 3.5, numpy 1.11.1,  and osx 10.11.6.

Thanks for any suggestions

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -19,6 +19,7 @@
 ### Bug fixes
 
 * `skbio.tree.TreeNode.prune` and implicitly `skbio.tree.TreeNode.shear` were not handling a situation in which a parent was validly removed during pruning operations as may happen if the resulting subtree does not include the root. Previously, an `AttributeError` would raise as `parent` would be `None` in this situation. 
+* numpy linking was fixed for installation under El Capitan.
 
 ### Deprecated functionality [stable]
 

--- a/setup.py
+++ b/setup.py
@@ -73,13 +73,16 @@ if platform.machine() == 'i686':
 
 extensions = [
     Extension("skbio.stats.__subsample",
-              ["skbio/stats/__subsample" + ext]),
+              ["skbio/stats/__subsample" + ext],
+              include_dirs=[np.get_include()]),
     Extension("skbio.alignment._ssw_wrapper",
               ["skbio/alignment/_ssw_wrapper" + ext,
                "skbio/alignment/_lib/ssw.c"],
-              extra_compile_args=ssw_extra_compile_args),
+              extra_compile_args=ssw_extra_compile_args,
+              include_dirs=[np.get_include()]),
     Extension("skbio.diversity._phylogenetic",
-              ["skbio/diversity/_phylogenetic" + ext])
+              ["skbio/diversity/_phylogenetic" + ext],
+              include_dirs=[np.get_include()])
 ]
 
 if USE_CYTHON:

